src/plugins/ResizeReorder/ResizerKnob.js

```diff
@@ -82,18 +88,56 @@ class ResizerKnob extends React.PureComponent {
     return (
       <div
         className={cx('fixedDataTableCellLayout/columnResizerContainer')}
-        ref={(element) => (this.curRef = element)}
+        ref={this.setResizerKnobRef}
         style={resizerKnobStyle}
-        onMouseDown={this.onMouseDown}
-        onTouchStart={this.props.touchEnabled ? this.onMouseDown : null}
-        onTouchEnd={this.props.touchEnabled ? this.suppressEvent : null}
-        onTouchMove={this.props.touchEnabled ? this.suppressEvent : null}
```

Comment:
Same reasoning as above. 
I moved the registration to `setupHandlers()` and deregistration to `cleanupHandlers`.